### PR TITLE
Added ingress hosts example for Artifactory chart

### PR DESCRIPTION
**Pull request by hoeghh**
ingress.hosts being an array, it makes sense to show how to set the value of it.

*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

### CODE REVIEW

**Comment by hoeghh** [comment]
signed the Contributor License Agreement

File: stable/artifactory/README.md [hunk 1]
```diff
@@ -144,7 +145,14 @@ The following tables lists the configurable parameters of the artifactory chart
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
 
-### Ingress TLS
+### Ingress and TLS
+To get Helm to create an ingress object with a hostname, add these two lines to your Helm command:
+```
+helm install --name artifactory \
+  --set ingress.enabled=true \
```
Comment (jainishshah17) [inline stable/artifactory/README.md:152]: Maybe worth adding --set ingress.tls.secretName="chart-example-tls" \

Comment (hoeghh) [reply stable/artifactory/README.md:152]: Yes I will add that later on. I think the doc need a lot of small changes to be easy to use. This small change is by request from a company that found it hard to figure out how the syntax for the hosts argument worked, so I added that.

**Comment by jainishshah17** [comment]
@hoeghh Can you bump chart version.

**Comment by hoeghh** [comment]
@jainishshah17 yes, i will do that doing the new week. I didnt know that was necessary for adding documentation, but it makes good sense.

**Comment by hoeghh** [comment]
@jainishshah17 I incremended the chart version, and the CI system still fails.
What did i do wrong?

**Comment by unguiculus** [comment]
Please merge in the latest master and you'll see. You need to bump again.

**Comment by hoeghh** [comment]
@jainishshah17 done :)

**Pull request merged**